AEPServices/Sources/ui/fullscreen/FullscreenMessage.swift

```diff
@@ -16,23 +16,32 @@ import WebKit
 
 /// This class is used to create and display fullscreen messages on the current view.
 @objc(AEPFullscreenMessage)
-public class FullscreenMessage: NSObject, WKNavigationDelegate, FullscreenPresentable {
+public class FullscreenMessage: NSObject, FullscreenPresentable {
```

Comment:
Can we move this implementation behind an interface? The direct dependency on WebKit in this class will make it difficult to port the changes over to tvOS.

Reply:
i agree this is a good change to make.  the dependency on WebKit didn't get added to this PR, so i'd think these changes should be made when we add support for tvOS.  

thoughts?